src/App.test.js

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
 import App from './App';
 
 it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+  render(<App />);
```

Comment:
Could create an assertion to test that `jest-dom` is working correctly.

Reply:
Good idea, thanks Alex 😊 I just pushed the test using it.